definitions/contested/json/CaseEventToFields/CaseEventToFields-nonprod.json

```diff
@@ -1,2 +1,14 @@
 [
+  {
+    "LiveFrom": "01/01/2021",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseEventID": "FR_draftOrderApproved",
+    "CaseFieldID": "latestDraftDirectionOrder",
```

Comment:
same as above, don't use generic nonprod files. I think your ticket should be toggled on.